Makefile

```diff
@@ -138,4 +138,17 @@ build-ci: .docker-build-pull
 test-ci: .docker-build-ci
 	${DC_CI} run test-image
 
-.PHONY: all clean build pull docs livedocs build-docs lint run stop kill run-shell shell test test-image rebuild build-ci test-ci fresh-data djshell run-prod build-prod test-cdn
+#########################
+# Requirements management
+#########################
+
+compile-requirements: .docker-build-pull
+	${DC} run --rm compile-requirements
+
+upgrade-requirements: .docker-build-pull
+	${DC} run --rm upgrade-requirements
+
+check-requirements: .docker-build-pull
+	${DC} run --rm test pip list -o
```

Comment:
These 3 should be added to the `help` output.

Reply:
- [ ] @stevejalim update makefile help output